packer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "variables": {
+    "aws_access_key": "",
+    "aws_secret_key": "",
+    "db_host": null,
+    "db_name": null,
+    "db_user": null,
+    "db_pass": null,
+    "subnet_id": null
+  },
+  "provisioners": [
+    {
+      "type": "ansible",
+      "playbook_file": "../ansible/wordpress.yml",
+      "extra_arguments": ["--extra-vars", "db_host={{user `db_host`}} db_name={{user `db_name`}} db_user={{user `db_user`}} db_pass={{user `db_pass`}}"]
+    }
+  ],
+  "builders": [{
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "us-east-1",
```

Comment:
Should be able to read this from the environment, since we set it for Terraform earlier in the process, yeah?
Something like:
```
  "region": "{{env `AWS_DEFAULT_REGION`}}"
```

Reply:
Good call!